shuup/admin/static_src/base/js/editor.js

```diff
@@ -64,6 +64,12 @@ function activateEditor($editor, attrs = {}) {
     $editor.parent().find(".note-editable").on("drop", (event) => {
         cancelEvent(event);
     });
+
+    $(".note-codable").on("blur", function() {
```

Comment:
I really didn't get what are you trying to achieve. If you want to invoke some function in summernote instance, you can already use `$editor` here.. and I am pretty sure you can access the textarea using `$editor.parent().find("textarea");` Please, take a look at the existing code.